### ISSUE TITLE
🐛 Bug Report: Error is shown, but user is still created

### ISSUE THREAD

**Issue by psschand16** [appwrite/playground-for-web#80]
### Account Registration and other api endpoints are not stable or not idempotant.

I am using the below cdn

```
<script src="https://cdn.jsdelivr.net/npm/appwrite@10.2.0" />
```

**on debugging I its failing below, which is in Web SDK in /appwrite/src/services/account.ts**

```

 const uri = new URL(this.client.config.endpoint + path);
            return await this.client.call('post', uri, {
                'content-type': 'application/json',
            }, payload);

```

<img width="878" alt="abc143 register successful Screenshot 2023-01-31 100020" src="https://user-images.githubusercontent.com/61744673/215740639-83755cf2-d474-4a92-8080-5c3438ef5603.png">
<img width="892" alt="abc144 failed to create account Screenshot 2023-01-31 100433" src="https://user-images.githubusercontent.com/61744673/215740684-4dba48f3-204f-4dd6-9f86-3cdb7327184f.png">
<img width="770" alt="appwrite console Screenshot 2023-01-31 100536" src="https://user-images.githubusercontent.com/61744673/215740714-e2727836-796c-4665-b1d7-01958c5d624c.png">

### 👍 Expected behavior

Ideally user is either created or not created.
users should be created as account api is called with necessary parameters

### 👎 Actual Behavior

I am getting different results for the same endpoint 
As an example in  the case of a register  register endpoint.

**case 1:  account created successfully** 
I recieve a popup that account is created successfully.

**case 2: account not created successfully but user is successfully entered in Appwrite auth console**
you get a popup user not created but you can find them appwrite console

**case 3:  account not created successfully but user is not found in Appwrite auth console**

### 🎲 Appwrite version

Version 1.2.x

### 💻 Operating system

Windows

### 🧱 Your Environment

web sdk
index.html on a live server

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by psschand16** [comment]
https://appwrite.omggn.tk/

**Comment by psschand16** [comment]
Hi @stnguyen90  Thanks looking into this .

please note you can test the same html/javascript app at the below link
https://appwrite.omggn.tk/

I am sharing screenshots of the network requests below, for that case 2.

![image](https://user-images.githubusercontent.com/61744673/216159522-8fee36d5-1509-4743-804c-d7942b049e2e.png)
![image](https://user-images.githubusercontent.com/61744673/216159575-906726aa-17ca-48cc-ac02-4fe17caf0952.png)
![image](https://user-images.githubusercontent.com/61744673/216159616-e86d8ae3-c5e0-4e04-be47-d42fdfbf5c6a.png)


![image](https://user-images.githubusercontent.com/61744673/216159684-d19375b1-1365-46bd-91fa-fa3cd430c84b.png)
![image](https://user-images.githubusercontent.com/61744673/216159712-57ed3315-5e0e-4f87-b647-b99cfc12aebe.png)
![image](https://user-images.githubusercontent.com/61744673/216159754-75e9a966-0b17-469e-aaf5-e1fb2eb4bf40.png)
![image](https://user-images.githubusercontent.com/61744673/216159860-1b709421-11a0-48a6-bb48-2971f4d9b53d.png)
![image](https://user-images.githubusercontent.com/61744673/216159898-18fe2a96-e30f-4cdc-9791-f2c5692a1ace.png)
![image](https://user-images.githubusercontent.com/61744673/216159930-7fc35fb5-891e-4858-af51-843d32c712d3.png)

**Comment by stnguyen90** [comment]
@psschand16, thanks for all the info! 🙏🏼 I think the form is submitting and canceling the network request. we should probably add some `event.preventDefault()` call or something.

**Comment by smilebasti** [comment]
Hi, 
would be great to see this implemented :)

@psschand16 have you found a solution or workaround?

**Comment by smilebasti** [comment]
If anybody else finds this issue, i fixed it with this merge request commit: #66 

See commit 1 [https://github.com/appwrite/playground-for-web/pull/69/commits/481e1c38200cdc63a2a01a45988d6312ec09fee0](https://github.com/appwrite/playground-for-web/pull/69/commits/481e1c38200cdc63a2a01a45988d6312ec09fee0)

**Comment by stnguyen90** [comment]
Fixed as part of https://github.com/appwrite/playground-for-web/pull/83
